src/Controls/Maps/src/AppHostBuilderExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Maui.Controls.Maps;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.LifecycleEvents;
+using Microsoft.Maui.Maps.Handlers;
+
+#if ANDROID
+using Android.Gms.Common;
+using Android.Gms.Maps;
+#endif
+
+namespace Microsoft.Maui.Controls.Hosting
+{
+	public static partial class AppHostBuilderExtensions
+	{
+		public static MauiAppBuilder UseMauiMaps(this MauiAppBuilder builder)
```

Comment:
Should we take in the Google Maps API key here? And/or check if it's setup right and throw an exception if it isnt?

Reply:
Is there a way to use the key , i think you need to have it on manifest

Reply:
Yeah that might be right... Maybe we can check if it's there and log an error or even throw an exception if it isn't in the manifest at all?